### ISSUE TITLE
getGPUInfo() error on Mac when using software rendering

### ISSUE THREAD

**Issue by robatwilliams** [electron/electron#17451]
* **Electron Version:**
  * 4.0.4
* **Operating System:**
  * macOS 10.14 (Mojave) on iMac
  * macOS 18.2.0 (Darwin) on 2018? MacBook Pro
* **Last Known Working Electron version:**:
  * n/a

### Expected Behavior
The `getGPUInfo()` call succeeds when hardware acceleration is disabled.

### Actual Behavior
The promise is rejected with an error: `Error: GPU access not allowed. Reason: GPU process crashed too many times with SwiftShader.`

The GPU process does in fact crash (only once); the `gpu-process-crashed` event is raised.

### To Reproduce
This *should* do it, but I don't have access to a Mac to try a standalone example:
1. download [electron-quick-start](https://github.com/electron/electron-quick-start)
2. add `app.getGPUInfo('complete').then(console.log);` in `main.js:createWindow`
3. `npm start -- --disable-gpu`
4. observe the output in the console

### Screenshots
n/a

### Additional Information
It probably does the same thing if the GPU is blacklisted, rather than explicitly disabled.

It probably does the same thing in `basic` mode too.

**Comment by codebytere** [comment]
@robatwilliams we don't manipulate GPU info as we pull it from native Chromium implementations, so this might not be the best place to report this. If you could please verify you see the same issues when  checking `chrome://gpu` with the same flags passed, this issue should closed and opened on crbug.com instead. Thanks!

**Comment by robatwilliams** [comment]
chrome://gpu reports correctly without crashing in the same circumstance.

**Comment by electron-triage** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron 6.x.y or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by electron-triage** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
